website/docs/src/content/cli-reference/env/show.mdx

```diff
@@ -0,0 +1,38 @@
+---
+title: gensx env show
+description: Show current GenSX environment details
+---
+
+# gensx env show
+
+The `gensx env show` command displays the name of the currently selected environment.
+
```

Comment:
Should this just be `gensx env` to show the currently selected env? This is fine to have as well I guess.